checker/check_result.go

```diff
@@ -63,6 +63,32 @@ const (
 	FileTypeURL
 )
 
+// OutcomeType represents the outcome of a result.
+type OutcomeType int
```

Comment:
Where is this being exposed?

Reply:
not used yet. I can remove it for this PR if you want. Shall I do that?

Reply:
removed.